lectures/inflation_history.md

```diff
@@ -167,9 +168,9 @@ for col in cols:
             y=df_fig5[col].iloc[-1], s=col)
 
 ax.set_yscale('log')
-ax.set_ylabel('Index  1913 = 100')
+ax.set_ylabel('Logged Price Levels (Index  1913 = 100)')
 ax.set_ylim([10, 1e6])
-ax.set_xlabel('Logs of price levels')
+ax.set_xlabel('Year')
```

Comment:
```suggestion
ax.set_xlabel('year')
```

[axis labels should be lower case (“time” not “Time”, etc.)](https://manual.quantecon.org/styleguide/figures.html)